toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/worklets/guest_start_nvme.py

```diff
@@ -182,6 +183,35 @@ def namespace(controller_id, nsid, aux={}):
     nvme += qemu_nvme_dev4
     drives.append(drive4)
 
+    #
+    # Nvme3 - Controller with FDP enabled subsystem
+    #
+    subsys_name = "subsys0"
+    subsys_attributes = {
+        "fdp": "on",
+        "fdp.nruh": "8",
+        "fdp.nrg": "32",
+        "fdp.runs": "40960",
+    }
+    nvme += subsystem(subsys_name, aux=subsys_attributes)
+
+    controller_id4 = "nvme3"
+    controller_bus4 = "pcie_downstream_port4"
+    controller_slot4 = 4
+    nvme += controller(
+        controller_id4,
+        "beefcace",
+        0,
+        controller_bus4,
+        upstream_bus,
+        controller_slot4,
+        subsys_name,
+    )
+
+    drv_fdp, qemu_nvme_dev_fdp = namespace(controller_id4, 1)
+    nvme += qemu_nvme_dev_fdp
+    drives.append(drv_fdp)
+
```

Comment:
not super important, but may be useful in future.

for the namespace, we can choose the reclaim unit handles to be associated with the namespace with fdp.ruhs, i.e. if we specify fdp.ruhs=0;5;6;7 we get 4 host-specified RUHs (0, 5, 6 and 7), if we do not, we get a single controller specified RUH.